api/tests/unit/organisations/chargebee/test_unit_chargebee_chargebee.py

```diff
@@ -44,12 +44,14 @@ def __init__(
         plan_id="plan-id",
         created_at=datetime.utcnow(),
         customer_id="customer-id",
+        chargebee_email="chargebee-email",
```

Comment:
this argument should really be `customer_email`

Reply:
OK fixed

Reply:
Actually fixed now...